lib/transformations/loaders/loaders.js

```diff
@@ -3,114 +3,171 @@ const safeTraverse = require('../safeTraverse');
 module.exports = function(j, ast) {
 
 	const createArrayExpression = function(p) {
-		var objs = p.parent.node.value.value.split('!').map(val => j.objectExpression([j.property('init', 
-										j.identifier('loader'), 
-										j.literal(val)
-										)]));
-		var loaderArray = j.arrayExpression(objs);
+		let objs = p.parent.node.value.value.split('!')
```

Comment:
Note - Let's move such references(which are necessary) to some common meaningful methods.